MoviesApp/MoviesApp/UIImageView+Extension.swift

```diff
@@ -0,0 +1,19 @@
+// UIImageView+Extension.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Расширение для получения изображения
+extension UIImageView {
```

Comment:
```suggestion
extension UIImageView {
```
 Это в сл дз вынести в имидж сервис